### DISCUSSION TITLE
How to run Query-to-file in my system

### DISCUSSION THREAD

**Discussion by JAgrit20** [polypheny/Polypheny-DB#328] [Q&A] [answered]
I have cloned the repo, How do I run on my Linux system?

**Comment by nilshansen94** [comment, marked as answer]
Hi Jagrit

The README.md on [github](https://github.com/polypheny/Query-to-File) has quite some instructions on how to get the Query-to-File running. First, you need to make sure to be working with Java 11. Furthermore, you need to install FUSE.  The [installation guide](https://github.com/SerCeMan/jnr-fuse/blob/master/INSTALLATION.md) of JNR-FUSE  contains useful information on how to install on different operating systems. 

After installing FUSE, you need to locate the FUSE binary (.dll / .dylib / .so) and specify the path in the config file (`Query-to-File/src/main/resources/config.properties`). There are already common paths and library names in there (see this [link](https://github.com/polypheny/Query-to-File/blob/main/src/main/resources/config.properties)). Make sure that there is only one entry for `libraryPath` and `libfuse`enabled in the config file.

The main-method is located in `src/main/java/org/polypheny/qtf/Main.java`. Please be aware that you need to have a running instance of Polypheny-DB.

**Comment by wamiljajoo** [comment]
Polypheny-DB should provide Apache FtpServer?
@nilshansen94 @vogti
